media-collection/TAGS_IMPL.md

```diff
@@ -0,0 +1,45 @@
+# Overview
+
+Customers that are using Adobe Tags to implement Adobe solutions, can implement Media using Web SDK Tags Extension.
+
+# Configuration
+
+To configure Media, when installing the Web SDK Extension, there is a Media section where settings like `player name`,
+`channel` have to be provided.
+![image](public/img/configuration.png)
+
+# Create a session
+
+To create a Media Session, similarly to the standalone library implementation, session details are needed as well as an
```

Comment:
We should add more details here around the `Rule` and how to hook up the event, in addition to the action.

Reply:
Also, we should change the action name in the extension from `Create Media Analytics Session` to `Create Media Session`

Reply:
Since the `Create Media Session` action requires the `XDM` field, we should explain how to populate that before the customer starts building the rule.

Reply:
It seems like the `On before media event send` callback is required, so we should add a `*` next to it, and maybe a commented out sample code in there.

Reply:
One last thing (I will move these comments to the code PRs):

I noticed that the `pings` kept going even after a session end event. Is there a specific event that stops the pings?

Reply:
yes, it's either `sessionEnd` or `sessionComplete`.